symphony-bdk-legacy/symphony-bdk-bot-sdk-java/build.gradle

```diff
@@ -1,5 +1,14 @@
+plugins {
```

Comment:
this plugin is already applied at top level, it is weird that we had to apply it here